### ISSUE TITLE
Don't works well without phpunit.xml & tests folder

### ISSUE THREAD

**Issue by nunomaduro** [pestphp/pest-plugin-init#5]

**Comment by owenvoke** [comment]
Yep, that's why I added https://github.com/pestphp/pest-plugin-init/pull/3 :+1:

However, I also ran into the `tests` directory issue. I guess that has to be solved in the core perhaps?

**Comment by nunomaduro** [comment]
Well, I think so, but would be cool to ignore that error on the core if the `init` option exist or something. Fell free to pull request that on the core.

**Comment by nunomaduro** [comment]
@owenvoke The fact that `tests` do not exist needs to be fixed on the core.
